src/Dashboard/App_Start/AppModule.cs

```diff
@@ -123,16 +123,40 @@ public override void Load()
             }
         }
 
+        // Optional Appsetting to explicitly set the table name to use with fast logging. 
+        const string FunctionLogTableAppSettingName = "AzureWebJobsLogTableName";
+
+        // Optional Appsetting to control site extension versioning. We can infer log mode form this. 
+        const string FunctionExtensionVersionAppSettingName = "FUNCTIONS_EXTENSION_VERSION";
+        const string FunctionExtensionVersionDisabled = "disabled";
+
         // Get a fast log table name 
         // OR return null to use traditional logging. 
         private static CloudTable TryGetLogTableName(CloudTableClient tableClient)
```

Comment:
This method is named incorrectly. It doesn't return a table name, but a CloudTable instance (or null). Ideally you'd have a method that resolves the name (which can then be testable) and you create a CloudTable from that.